bootstrapper/initproto/init.proto

```diff
@@ -10,14 +10,15 @@ service API {
 
 // InitRequest is the rpc message sent to the Constellation bootstrapper to initiate the cluster bootstrapping.
 message InitRequest {
+  reserved 4;
+  reserved "cloud_service_account_uri";
   // KmsUri is an URI encoding access to the KMS service or master secret.
   string kms_uri = 1;
   // StorageUri is an URI encoding access to the storage service.
   string storage_uri = 2;
   // MeasurementSalt is a salt used to generate the clusterID for the initial bootstrapping node.
   bytes measurement_salt = 3;
-  // CloudServiceAccountUri is an URI encoding the cloud service account.
-  string cloud_service_account_uri = 4;
+  // string cloud_service_account_uri = 4; removed
```

Comment:
I don't think we should keep this line it if we proceed as stated in their docs, but remove it entirely.